serverless.yml

```diff
@@ -0,0 +1,131 @@
+service: cvs-svc-technical-records
+
+openapi: 3.0.0
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+
+plugins:
+  - serverless-dynamodb-local
+  - serverless-offline
+  - serverless-openapi-documentation
+
+custom:
+  serverless-offline:
+    port: 3004
+  dynamodb:
+    start:
+      port: 8002
+      inMemory: true
+      migrate: true
+      seed: true
+    seed:
+      tech-records:
+        sources:
+          - table: cvs-local-tech-records
```

Comment:
please agree to a consistency between "tech-records" and "technical-records" throughout the project